packages/trader/src/App/Components/Layout/Footer/toggle-settings.jsx

```diff
@@ -76,13 +88,15 @@ const ToggleSettings = ({
             <React.Suspense fallback={<UILoader />}>
                 <Modal
                     id='dt_settings_modal'
-                    modal_content={modal_content}
```

Comment:
@mahdipourziaei-binary we used to send content as a prop. now i'm rendering the `vetical-tab` outside of modal and send it as childre

Reply:
@Mahboobeh-binary That's fine, I'm just referring to the variable `content` on line 39.

Reply:
@mahdipourziaei-binary it is the same variable. i havent changed that